### PR TITLE
Combine workflwos into one

### PR DESCRIPTION
**Pull request by EpicWink**
Put all wheel-build CI jobs into one workflow. This makes it easier to update in the future.

At the same time, I added PIP dependency wheel caching to speed up jobs (see [the documentation](https://pip.pypa.io/en/stable/user_guide/#installing-from-local-packages)), and updated the build badge to point to the Github Actions workflow (easy not that there's only one workflow, but you can add a badge for each new workflow in the future)

### CODE REVIEW

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -30,13 +54,15 @@ jobs:
         # Skip building on Python 2.7 and PyPy
         # Additionally, skip 32-bit Windows for now as MSVC needs seperate setup with different toolchain to do this
         # Refer: https://cibuildwheel.readthedocs.io/en/stable/cpp_standards/#windows-and-python-27
-        CIBW_SKIP: cp27-* pp* *-win32
+        CIBW_SKIP: cp27-* pp* ${{ matrix.cibw_skip_extra }}
```
Comment (chaitan94) [inline .github/workflows/wheels.yml:57]: Actually there is no need for an extra `cibw_skip_extra` variable. `*-win32` would only be present in `windows-latest` image only, it doesn't affect the others.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # pytomlpp
 
-[![Build Status](https://travis-ci.com/bobfang1992/pytomlpp.svg?branch=master)](https://travis-ci.com/bobfang1992/pytomlpp)
+[![Build Status](https://github.com/bobfang1992/pytomlpp/workflows/Wheels/badge.svg)](https://travis-ci.com/bobfang1992/pytomlpp)
```
Comment (bobfang1992) [inline README.md:3]: Could we also change the fallback url from Travis to Github Action? Thanks.

Comment (EpicWink) [reply README.md:3]: Not actually a fallback URL, rather the link the user is taken to when they click on it

**Comment by bobfang1992** [comment]
LGTM, feel free to merge if you are ready!

**Comment by bobfang1992** [comment]
Hey looking at the github action page now it seems the CI job for windows is not triggered at all at this moment. Do you know why that's the case? Thanks @EpicWink

**Comment by EpicWink** [comment]
I accidentally deleted the Windows matrix entry. Fixed in master. To add the Windows build to the 0.2 PyPI release, download the dist artifact and upload the Windows wheels using Twine

**Pull request merged**